pcdsdevices/valve.py

```diff
@@ -227,6 +227,34 @@ class VGC(VRC):
                     doc=('individual valve MPS state for debugging'))
 
 
+class VFS(Device):
+    """Class for Fast Shutter Valve."""
+    request_close = Cpt(EpicsSignalWithRBV, ':CLS_SW', kind='normal',
+                        doc='Request Fast Shutter to Close')
+    request_open = Cpt(EpicsSignalWithRBV, ':OPEN_SW', kind='normal',
+                       doc='Request Fast Shutter to Open')
```

Comment:
There's nothing wrong with how you've set these up, but in the other valve classes we call these `close_command` and `open_command`. You can decide if it's better for this device to look the same, or if it is better to keep it looking different because it is different.

Reply:
Made updates to the doc string. Will keep the current names for now.